docs/source/tutorial/tutorial_2.mdx

```diff
@@ -239,35 +249,37 @@ override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
 
 ## Adding more info to the list
 
-First, take advantage of one of the built-in styles for `UITableViewCell` to display more information without having to do much work. Go to **Main.storyboard**, select the **Master Scene** which includes the TableView and the default cell. 
+Let's take advantage of one of the built-in styles for `UITableViewCell` to display more information without having to do much work. Go to `Main.storyboard`, select the **Master Scene** which includes the `TableView` and the default cell. 
 
-Open the flippy triangles in the left sidebar of interface builder until you get to **Cell**. Select that element, then go to the **Attributes Inspector** in the right sidebar of Xcode. At the top, there's a drop-down which will allow you to select a style for your table view cell. Select **Subtitle**:
+Expand the dropdowns in the left sidebar of Interface Builder until you get to **Cell**. Select that element, then go to the **Attributes Inspector** in the right sidebar of Xcode. At the top, there's a drop-down that allows you to select a style for your table view cell. Select **Subtitle**:
 
 ![](images/use_subtitle_style.png)
 
-Also, while you're here, this is a good time to give the main TableView a name that better reflects what it's actually showing. Select the navigation item and rename it from `Master` to `Launches`:
+While you're here, this is a good time to give the main `TableView` a name that better reflects what it's actually showing. Select the navigation item and rename it from `Master` to `Launches`:
 
 ![](images/rename_master_launches.png)
 
-Next, go back to **LaunchList.graphql**. You've already got most of the information you want to display, but it would be nice to display both the name of the mission and an image of the patch. 
+Go back to `LaunchList.graphql`. Your query is already fetching most of the information you want to display, but it would be nice to display both the name of the mission and an image of the patch. 
 
-Looking at the schema in GraphiQL, you can see that `Launch` has a property of `mission`, which will allow you to get details of the mission. A mission has both a name and a `missionPatch` property, and the `missionPatch` can optionally take a parameter about what size something needs to be.
+Looking at the schema in GraphiQL, you can see that `Launch` has a property of `mission`, which allows you to get details of the mission. A mission has both a name and a `missionPatch` property, and the `missionPatch` can optionally take a parameter about what size something needs to be.
 
-Since loading a TableView with large images can be a real performance hog, ask for the name and a small mission patch. Update your query to look like the following:
+Since loading a `TableView` with large images can be a real performance hog, ask for the name and a small mission patch. Update your query to look like the following:
```

Comment:
The `TableView` here either shouldn't be backticked or should be `UITableView` - it could also be `table view`